### DISCUSSION TITLE
Installing other ai models for use with Krita Diffusion

### DISCUSSION THREAD

**Discussion by itchymitchy** [Acly/krita-ai-diffusion#828] [Q&A]
Can you add other ai models into Krita once you have the basic set-up running? I was watching a YouTube video by Streamtabulous, and he has a bunch of ai models in his AI image generation drop down drop menu.

**Comment by freecoderwaifu** [comment]
Download them from Civitai then put them in their respective folders inside the Comfy UI models folder: checkpoints or loras. The models folder is inside your respective Comfy UI folder, either the built in install or a standalone install.

The built in one is by default in AppData\Roaming\krita\ai_diffusion\server, you can find the location in Settings > Connection
